packages/@aws-cdk/toolkit/lib/api/io/private/logger.ts

```diff
@@ -24,6 +24,27 @@ export function withAction(ioHost: IIoHost, action: ToolkitAction) {
   };
 }
 
+export function withoutColor(ioHost: IIoHost): IIoHost {
```

Comment:
not sure about how we feel about this but it currently replaces style as well, i.e. `chalk.bold`

Reply:
Yeah that's fine